.github/ISSUE_TEMPLATE/90_bug_report.yml

```diff
@@ -0,0 +1,104 @@
+name: Bug report
+description: File a bug report. If you need help, please join our [Discord server](https://discord.com/invite/ollama).
```

Comment:
The link isn't being rendered correctly. Maybe it's a raw string?

![image](https://github.com/ollama/ollama/assets/2372640/d99e0ebd-c1c9-487e-ac4d-700b87a9c146)